src/ctapipe/utils/astro.py

```diff
@@ -4,10 +4,151 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from pathlib import Path
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
+from astroquery.vizier import Vizier
```

Comment:
This is a new dependency. It should be included in pyproject.
@maxnoe is it ok to have it? Or we need to download and stash a cached version of the catalog?

Reply:
I'd prefer having the catalog we need as a resource file, it should be fairly small.

It would be good have the code to create it here in any case, but that can be done without hard-requiring astroquery. it should be kept as an optional dependency.

Reply:
i made a function to make the catalog file in the last commit and added astroquery to the optional dependencies. 

The function get_bright_stars() should then only look to the resource file with the function get_table_dataset(), no? 
Schould we then have multiple files uploaded for different catalogues?

Reply:
Yes. And only if your really need multiple catalogs.

I guess we should keep the yale bright stars for compatibilty. If you require another catalog for your applications, select the most appropriate one and add it.

I.e. after this PR, `get_bright_stars()` should still return the stars of the yale bright star catalog, but with the correct positions after applying the space motion.
Add a second function for the catalog you choose, e.g. `get_hipparcos_stars()`.